package.json

```diff
@@ -3,9 +3,9 @@
   "version": "1.6.0",
   "description": "A RESTful API for submitting payments and monitoring accounts on the Ripple network.",
   "scripts": {
-    "test": "NODE_ENV=test ./node_modules/.bin/istanbul test ./node_modules/.bin/_mocha -- --ui tdd --reporter spec --timeout 10000 --slow 500 test/*-test.js test/unit/*-test.js",
+    "test": "NODE_ENV=test istanbul test _mocha",
```

Comment:
How can you leave off `./node_modules/.bin` without having these packages installed globally?